### DISCUSSION TITLE
SSL pinning in Nativescript Angular app

### DISCUSSION THREAD

**Discussion by KMalaimegam** [NativeScript/NativeScript#10549] [Show and tell]
Hello Team,

We’re interested in implementing SSL pinning in angular app. However, we haven’t come across any relevant libraries or events. Could you please advise if there’s a way to implement this or if there are any suitable plugins available?

This plugin **@nativescript-community/https** does not work SSL pinning. Even there are open bugs related to SSL spinning.

**Comment by insytes** [comment]
Hi @KMalaimegam 👋🏻 

SSL pinning isn't supported in `@nativescript/core` but as you mentioned already, it is supported in `@nativescript-community/https`

Would you be able to create an issue in that repo describing exactly what you have tried and what error you are getting?
